client/app/home/translations/Messages_fr_FR.json

```diff
@@ -5,5 +5,6 @@
   "homepage_type_of_guide_vps": "VPS",
   "homepage_type_of_guide_pci": "Public Cloud",
   "homepage_scheduled_maintenance_operation": "Une opération de maintenance est programmée pour la région BHS1 le 19 septembre 2018 entre 3:00 et 8:00 UTC-4. Aucune perturbation n'est à prévoir sur vos instances, seules certaines APIs de cette région seront injoignables pendant la durée de l'opération",
-  "homepage_scheduled_maintenance_operation_more_detail": "Plus de détail sur la tâche travaux correspondante:"
-}
\ No newline at end of file
+  "homepage_scheduled_maintenance_operation_more_detail": "Plus de détail sur la tâche travaux correspondante:",
```

Comment:
```suggestion
  "homepage_scheduled_maintenance_operation_more_detail": "Plus de détail sur la tâche travaux correspondante :",
```

Reply:
Hi got a specific request to add this space, can you please refer MBE-247. Even I prefer not to have space.

Reply:
@radireddy Yes you can safely add this space. It's a french grammar rule;)